### PR TITLE
Add restart options for cloudcore.service and edgecore.service

### PR DESCRIPTION
**Pull request by YaozhongZhang**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:
Add restart options for cloudcore.service and edgecore.service, so that they can restart automatically on unexpected exit.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by daixiang0** [comment]
lgtm 
ping @fisherxu

File: build/tools/cloudcore.service [hunk 1]
```diff
@@ -4,6 +4,8 @@ Description=cloudcore.service
 [Service]
 Type=simple
 ExecStart=/etc/kubeedge/cloudcore
+Restart=always
```
Comment (chendave) [inline build/tools/cloudcore.service:7]: Is that meaning service will always try when the exception cannot get recovered?

Comment (YaozhongZhang) [reply build/tools/cloudcore.service:7]: This "always" means that the service will be restarted regardless of whether the exit code is 0 or not.

Comment (YaozhongZhang) [reply build/tools/cloudcore.service:7]: I don't think it's a problem. And kubelet dose the same.

> [Unit]
> Description=kubelet: The Kubernetes Node Agent
> Documentation=https://kubernetes.io/docs/
> 
> [Service]
> ExecStart=/usr/bin/kubelet
> Restart=always
> StartLimitInterval=0
> RestartSec=10
> KillMode=process
> 
> [Install]
> WantedBy=multi-user.target

Comment (kevin-wangzefeng) [reply build/tools/cloudcore.service:7]: I'm fine with this

**Pull request merged**